### PR TITLE
Remove usage of catchsegv as it is not killed properly in case of timeout

### PR DESCRIPTION
**Pull request by hevrard**
Do not try to get the stack trace on Linux. We will revisit this later, see #338 for the discussion.

### CODE REVIEW

**Comment by afd** [comment]
I will look, but would prefer @paulthomson to do an approving review.

File: python/src/main/python/drivers/runspv.py [hunk 1]
```diff
@@ -231,13 +237,24 @@ def spirvopt_path():
...
-        # Didn't find catchsegv on path; that's OK
-        pass
+# FIXME: using gdb wrapping leads to a weird issue where, once a test times out,
+# subsequent tests leads to the whole worker being SIGSTOP'ed.
+# See https://github.com/google/graphicsfuzz/issues/338
+
+# def maybe_add_gdb(cmd: List[str]) -> None:
+#     # If gdb is available, use it to obtain backtraces in case of segfault
+
+#     # TODO: using gdb as a wrapper adds at least a second-long overhead to each
```
Comment (afd) [inline python/src/main/python/drivers/runspv.py]: Can you change TODO to TODO(issue number) where 'issue number' is the number of the associated issue for this optimization idea?

Comment (afd) [reply python/src/main/python/drivers/runspv.py]: (I like the reference to the issue above for the FIXME.  But this sound like it deserves a different issue.)

Comment (paulthomson) [reply python/src/main/python/drivers/runspv.py]: I propose we make this PR:

* Remove use of `catchsegv`
* Fix issue with subprocess_helper timeout

And we don't commit the commented out code; we instead create an issue, and put the code there or leave the code in a branch.

Comment (hevrard) [reply python/src/main/python/drivers/runspv.py]: See #349 for the fix of timeout.

Comment (paulthomson) [reply python/src/main/python/drivers/runspv.py]: Looks good. Let's make this just:

* Remove use of `catchsegv`

**Pull request merged**